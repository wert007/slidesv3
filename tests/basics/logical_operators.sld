func main() {
    print('false && loudlyFalse()');
    print(false && loudlyFalse());
    print('true || loudlyTrue()');
    print(true || loudlyTrue());
    print('true && loudlyFalse()');
    print(true && loudlyFalse());
    print('false || loudlyTrue()');
    print(false || loudlyTrue());
}

func loudlyFalse() -> bool {
    print('FALSE');
    return false;
}

func loudlyTrue() -> bool {
    print('TRUE');
    return true;
}