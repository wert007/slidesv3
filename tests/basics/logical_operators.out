false && loudlyFalse()
false
true || loudlyTrue()
true
true && loudlyFalse()
FALSE
false
false || loudlyTrue()
TRUE
true
= 0
